.github/ISSUE_TEMPLATE/aws-access-request.yml

```diff
@@ -125,11 +125,11 @@ body:
         ---
         :heavy_minus_sign::heavy_minus_sign::heavy_minus_sign::heavy_minus_sign: All done! :heavy_minus_sign::heavy_minus_sign::heavy_minus_sign::heavy_minus_sign:
         For SOCKS access, open a [SOCKS Access Request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=external-request%2Coperations%2Cops-access-request&template=socks-access-request.yml&title=SOCKS+access+for+%5Bindividual%5D).
-        If you need additional infrastructure, please use [#vfs-platform-support](https://dsva.slack.com/archives/CBU0KDSB1) 
+        If you need additional infrastructure, please use [#vfs-platform-support](https://dsva.slack.com/archives/CBU0KDSB1)
         :x: Don't fill out anything below here :x:
   - type: checkboxes
     attributes:
-      label: User must exist in a roster before AWS access can be granted
+      label: "User must exist in a roster before AWS access can be granted. See [AWS access documentation](https://vfs.atlassian.net/wiki/spaces/AMT/pages/2251358357/SOCKS+and+AWS+Access+Request+Automation#AWS-Access-Request) for the steps involved."
```

Comment:
The docs linked here aren't maintained anymore :( 

What do you think about linking this instead? https://vfs.atlassian.net/wiki/spaces/OT/pages/1792114735/Onboard+New+Team+Members+Granting+Access+to+Internal+Tools#AWS-Access-Request

Reply:
✅ Thanks!

Reply:
Actually, we also have this document https://vfs.atlassian.net/wiki/spaces/KB/pages/1949597798/AWS+Access+Add+User+or+Change+User+s+Permissions. Is it deprecated as well?

Reply:
wow 😂 I don't remember ever seeing that one before! I didn't even know there was a Space called "Knowledge base"!

It's generally accurate, but there have been some changes since that page was last updated. Honestly we could probably delete it since all the info in that page is in that first one I linked to.